### DISCUSSION TITLE
Mapping not being created for index on version 7.1.5

### DISCUSSION THREAD

**Discussion by akshaykatyal** [ruflin/Elastica#2098] [Q&A]
Hello, 
I am creating a index but while setting a mapping to the index in version 7.1.5 it gives me blank in the index mapping. The same thing works with version 6.2.1 where gettype is used but with version 7.1.5 the gettype is depreciated so it gives null response neither any error.
Any solution to this with verison 7.x in ruflin/elastica?
![Screenshot from 2022-07-04 10-29-12](https://user-images.githubusercontent.com/41782285/177115403-e08e622f-983c-4aa8-8e74-82e386e8b8e9.png)

**Comment by ruflin** [comment]
What does `setMapping` return on 7.1.5? Any error you see in the Elasticsearch logs?
